src/rollback_to_stable/rts_api.c

```diff
@@ -194,12 +204,17 @@ __rollback_to_stable(WT_SESSION_IMPL *session, const char *cfg[], bool no_ckpt)
 
     S2C(session)->rts->dryrun = dryrun;
 
+    __wt_timer_start(session, &timer);
+
     WT_STAT_CONN_SET(session, txn_rollback_to_stable_running, 1);
     WT_WITH_CHECKPOINT_LOCK(
       session, WT_WITH_SCHEMA_LOCK(session, ret = __rollback_to_stable_int(session, no_ckpt)));
 
+    /* Time since the RTS started. */
+    __wt_timer_evaluate(session, &timer, &time_diff);
     __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-      WT_RTS_VERB_TAG_END "finished rollback to stable%s", dryrun ? " dryrun" : "");
+      WT_RTS_VERB_TAG_END "finished rollback to stable%s and has ran for %" PRIu64 " seconds",
```

Comment:
There is a mistake in the original PR to develop that have to change it into milliseconds like others, but missed. Instead of adding overhead to backport two fixes, how about fixing it in this backport and fixing it in the develop using a separate ticket?

Reply:
done [43412ee](https://github.com/wiredtiger/wiredtiger/pull/9710/commits/43412ee632cb94a4c50325a35b443538e193b6bf)